scripts/hail_batch/hgdp1kg_tobwgs_densify_new_variants/hgdp_1kg_tob_wgs_densify_new_variants.py

```diff
@@ -0,0 +1,45 @@
+"""
+Densify TOB-WGS data.
+"""
+
+import click
+import hail as hl
+from analysis_runner import bucket_path, output_path
+
+
+NEW_VARIANTS = bucket_path(
+    'tob_wgs_hgdp_1kg_variant_selection/v8/tob_wgs_hgdp_1kg_filtered_variants.mt/'
+)
+GNOMAD_HGDP_1KG_MT = (
+    'gs://gcp-public-data--gnomad/release/3.1/mt/genomes/'
+    'gnomad.genomes.v3.1.hgdp_1kg_subset_dense.mt'
+)
+TOB_WGS = bucket_path('mt/v4.mt//')
+
+
+@click.command()
+def query():  # pylint: disable=too-many-locals
+    """Query script entry point."""
+
+    hl.init(default_reference='GRCh38')
+
+    hgdp_1kg = hl.read_matrix_table(GNOMAD_HGDP_1KG_MT)
+    tob_wgs = hl.read_matrix_table(TOB_WGS).key_rows_by('locus', 'alleles')
```

Comment:
Do we still need the `key_rows_by` here?

Reply:
I deleted it